lib/web_ui/test/engine/window_test.dart

```diff
@@ -463,6 +463,39 @@ Future<void> testMain() async {
     await expectLater(completer.future, completes);
   });
 
+  test('sets global html attributes', () {
+    final DomElement host = createDomHTMLDivElement();
+    final EngineFlutterView view = EngineFlutterView(dispatcher, host);
+
+    expect(host.getAttribute('flt-renderer'), 'html (requested explicitly)');
```

Comment:
Doesn't this break when the test runs in canvaskit mode?

Reply:
Test under `test/engine/` only run in html mode (for now).